groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -2833,31 +2740,18 @@ class optional {
 #ifdef BSLS_COMPILERFEATURES_SUPPORT_RVALUE_REFERENCES
     template <class ANY_TYPE>
     BSLSTL_OPTIONAL_ENABLE_ASSIGN_FROM_OPTIONAL_RVAL& operator=(
-                                                      optional<ANY_TYPE>&& rhs)
-        // If specified 'rhs' is engaged, assign its value to this object.
-        // Otherwise, reset this object to a disengaged state.  Return a
-        // reference providing modifiable access to this object.  Note that
-        // this method may invoke assignment from 'rhs', or construction from
-        // 'rhs', depending on whether this 'optional' object is engaged.
-    {
-        // Must be in-place inline because the use of 'enable_if' will
-        // otherwise break the MSVC 2010 compiler.
-        if (rhs.has_value()) {
-            if (this->has_value()) {
-                this->value() = MoveUtil::move(rhs.value());
-            }
-            else {
-                this->emplace(MoveUtil::move(rhs.value()));
-            }
-        }
-        else {
-            this->reset();
-        }
-        return *this;
-    }
+                                                     optional<ANY_TYPE>&& rhs);
+        // Disengage this object if the specified 'rhs' object is disengaged,
+        // and move assign to this object the value of 'rhs.value()' (of
+        // 'ANY_TYPE') converted to 'TYPE' otherwise.  Return a reference
+        // providing modifiable access to this object.  Note that this method
+        // does not  participate in overload resolution unless 'TYPE and
```

Comment:
```suggestion
        // does not participate in overload resolution unless 'TYPE' and
```
Please check for other instances of missing `'`.  Also, there are other places where double-space is present.  I tried to point them out if I noticed them.

Reply:
done. regex handled the two spaces issue.